### ISSUE TITLE
Warning in can_dbc_parser

### ISSUE THREAD

**Issue by fred-apex-ai** [NewEagleRaptor/raptor-dbw-ros2#7]
I came across this warning in [this Autoware.Auto CI job](https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/jobs/1388943419)

```
--- stderr: can_dbc_parser
/builds/autowarefoundation/autoware.auto/AutowareAuto/src/external/new_eagle_raptor/can_dbc_parser/src/DbcMessage.cpp: In member function ‘void NewEagle::DbcMessage::SetFrame(can_msgs::msg::Frame_<std::allocator<void> >::SharedPtr)’:
/builds/autowarefoundation/autoware.auto/AutowareAuto/src/external/new_eagle_raptor/can_dbc_parser/src/DbcMessage.cpp:168:33: warning: ‘muxSwitch’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  168 |         if (muxSwitch->GetResult() == it->second.GetMultiplexerSwitch()) {
      |             ~~~~~~~~~~~~~~~~~~~~^~
/builds/autowarefoundation/autoware.auto/AutowareAuto/src/external/new_eagle_raptor/can_dbc_parser/src/DbcMessage.cpp: In member function ‘can_msgs::msg::Frame NewEagle::DbcMessage::GetFrame()’:
/builds/autowarefoundation/autoware.auto/AutowareAuto/src/external/new_eagle_raptor/can_dbc_parser/src/DbcMessage.cpp:122:33: warning: ‘muxSwitch’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  122 |         if (muxSwitch->GetResult() == it->second.GetMultiplexerSwitch()) {
      |             ~~~~~~~~~~~~~~~~~~~~^~
---
```

Please fix it and let us know when fixed so we can update our dependency in https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/blob/master/autoware.auto.foxy.repos . Or has this warning been fixed already and we should just upgrade?

Thanks!

**Comment by NewEagleRaptor** [comment]
Thanks for pointing this out! I'm working on a fix right now, & will release it as soon as it's tested.

**Comment by NewEagleRaptor** [comment]
Thanks for your patience. Fix is released as foxy-1.1.1.

**Comment by NewEagleRaptor** [comment]
Merge request made to Autoware to update version of raptor-dbw-ros2:
https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/merge_requests/1094
